functions/__tests__/expressExample/resend/index.test.js

```diff
@@ -10,24 +11,64 @@ testConfig.mockConfig({
     issuer: "issuer",
     jwk_uri: "some uri",
   },
+  api: {
+    api_token: "sometoken",
+    url: "fake_url",
+  },
 });
 
 jest.mock("../../../utils/middleware");
+jest.mock("axios", () => jest.fn());
 
 describe("Supertest", () => {
   afterEach(() => {
     jwtCheck.mockClear();
+    axios.mockClear();
+  });
+
+  it("Unverified User Should Get Authentication Email", async (done) => {
```

Comment:
When using `it` for testing, usually the message that follows should be able to be read as a sentence. For example, right now it reads as "It unverified users should get authentication email". A better way to write this is "It should send an Authentication Email to a User" or "It should return a 201 status".